docs/azure_arc_jumpstart/azure_arc_k8s/aks_iot_edge/aks_iot_edge_terraform/_index.md

```diff
@@ -0,0 +1,232 @@
+
+    ```shell
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcK8s" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcK8s",
+    "name": "http://AzureArcK8s",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/azure/role-based-access-control/best-practices)**
+
+* [Enable subscription with](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider) the two resource providers for Azure Arc enabled Kubernetes. Registration is an asynchronous process, and registration may take approximately 10 minutes.
+
+  ```shell
+  az provider register --namespace Microsoft.Kubernetes
+  az provider register --namespace Microsoft.KubernetesConfiguration
+  az extension add --name connectedk8s
+  az extension add --name k8sconfiguration
+  ```
+
+  You can monitor the registration process with the following commands:
+
+  ```shell
+  az provider show -n Microsoft.Kubernetes -o table
+  az provider show -n Microsoft.KubernetesConfiguration -o table
+  ```
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+* In the first script ( [edge_azure_vm.sh](https://github.com/Manuss20/Kubernetes_edge/blob/main/deploy/scripts/edge/edge_azure_vm.sh) ) that you use specifically to provision the necessary components in the VM to be able to deploy our "simulated" edge device:
+
+  * Download Install the required tools moby-engine
+  * Download & install the Azure aziot-edge
+  * Creation of a new configuration file for aziot-edge (/etc/aziot/config.toml)
+
+* In this second bash script ( [az_k8sconfig_helm_aks.sh](https://github.com/Manuss20/Kubernetes_edge/blob/main/deploy/scripts/helm/az_k8sconfig_helm_aks.sh) ) it will allow us to deploy our IoT Edge solution for AKS, configure our and associate our AKS cluster with Azure Arc, for this: 
+  * Log in to Azure with Service Principal & Getting AKS credentials (kubeconfig)
+  * Associate our AKS with Azure Arc
+  * Create Namespace iotedge in AKS
+  * Generate secret that contains the connectionstring of our edge device.
+  * Create Cluster-level GitOps-Config for deploying IoT Edge workload
```

Comment:
```suggestion
* In the first script ( [edge_azure_vm.sh](https://github.com/main/Kubernetes_edge/blob/main/deploy/scripts/edge/edge_azure_vm.sh) ) that you use specifically to provision the necessary components in the VM to be able to deploy our "simulated" edge device:

  * Download Install the required tools moby-engine
  * Download & install the Azure _aziot-edge_
  * Creation of a new configuration file for aziot-edge (/etc/aziot/config.toml)

* In this second bash script ( [az_k8sconfig_helm_aks.sh](https://github.com/main/Kubernetes_edge/blob/main/deploy/scripts/helm/az_k8sconfig_helm_aks.sh) ) it will allow us to deploy our IoT Edge solution for AKS, configure our and associate our AKS cluster with Azure Arc, for this: 
  * Log in to Azure with Service Principal & Getting AKS credentials (kubeconfig)
  * Associate our AKS with Azure Arc
  * Create Namespace _iotedge_ in AKS
  * Generate a secret that contains the connection string of our edge device.
  * Create Cluster-level GitOps-Config for deploying IoT Edge workload
```